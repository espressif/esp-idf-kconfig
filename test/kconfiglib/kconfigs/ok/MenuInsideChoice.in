mainmenu "Menu inside choice"

choice DRIVE_IN_USE
    prompt "Choose drive type"

    config WARP_DRIVE
        bool "Warp drive"

    config SUBLIGHT_DRIVE
        bool "Sublight drive"

    menu "Warp drive configuration"
        # This menu will be shown inside the choice in GUI/TUI,
        # but it is separated from the choice logic (WARP_SPEED is not a choice config).
        visible if WARP_DRIVE

        config WARP_SPEED
            int "Light years per second"
            default 8

    endmenu
endchoice
