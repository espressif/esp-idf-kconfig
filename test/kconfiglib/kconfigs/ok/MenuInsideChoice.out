#
# Automatically generated file. DO NOT EDIT.
# Espressif IoT Development Framework (ESP-IDF)  Project Configuration
#
CONFIG_WARP_DRIVE=y
# CONFIG_SUBLIGHT_DRIVE is not set

#
# Warp drive configuration
#
CONFIG_WARP_SPEED=8
# end of Warp drive configuration
